Dockerfile

```diff
@@ -9,4 +9,16 @@ RUN mkdir /usr/share/jenkins && \
          -L -o /usr/share/jenkins/jenkins.war
 EXPOSE 8080
 ENV JENKINS_HOME /var/lib/jenkins
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+
+ARG user=jenkins
+ARG group=jenkins
+ARG uid=1000
+ARG gid=1000
+RUN groupadd -g ${gid} ${group} \
+    && useradd -d "$JENKINS_HOME" -u ${uid} -g ${gid} -m -s /bin/bash ${user}
+
+RUN mkdir -p $JENKINS_HOME/init.groovy.d
+RUN chown -R jenkins:jenkins $JENKINS_HOME
```

Comment:
nit: since you defined the variables it seems like you would want to substitute ${user}:${group} for jenkins:jenkins